man/git-fork.md

```diff
@@ -21,9 +21,9 @@ git-fork(1) -- Fork a repo on github
 
   Remotes will use ssh if you have it configured with GitHub, if not, https will be used.
 
-  Create pull request for a project on GitHub via command line.
+  Create a fork of a project on GitHub via command line.
   
-  A personal access token is required for making the API call to open the pull request(s) in GitHub. [API Documentation here](https://docs.github.com/en/rest/reference/pulls#create-a-pull-request)
+  A personal access token is required for making the API call to create a fork in GitHub. [API Documentation here](https://docs.github.com/en/rest/reference/pulls#create-a-pull-request)
```

Comment:
The link should be updated.

Reply:
Sorry, missed that.  Thanks for keeping a good eye on the PRs.  Fixed this via 30383f9